test/fixtures/gometalinter/code/pass/mypkg/mypkg.go

```diff
@@ -1,6 +1,14 @@
 package mypkg
 
+import "os/user"
+
 // World returns a string World.
 func World() string {
-	return "World"
+	user, err := user.Current();
+
+	if err != nil {
+		return "Could not get current user!"
```

Comment:
@antonlindstrom (off topic): What's the go way to handle this case? Return `err.String()` or something like that?